src/main/java/seedu/address/logic/parser/ClientFindCommandParser.java

```diff
@@ -14,9 +14,10 @@
 public class ClientFindCommandParser implements Parser<ClientFindCommand> {
 
     /**
-     * Parses the given {@code String} of arguments in the context of the ClientFindCommand
-     * and returns a ClientFindCommand object for execution.
-     * @throws ParseException if the user input does not conform the expected format
+     * Parses the given {@code args} in the context of the ClientFindCommand and returns a ClientFindCommand
+     * object for execution.
+     *
+     * @throws ParseException If the user input does not conform the expected format.
```

Comment:
```suggestion
     * @throws ParseException If the user input does not conform to the expected format.
```

Reply:
good catch, ill change the rest of the parsers as well